stacker/plan.py

```diff
@@ -81,7 +81,7 @@ def set_status(self, status):
         """
         if not isinstance(status, Status):
             raise ValueError(
-                "Step run_func must return a valid Status object. "
+                "Step status must return a valid Status object. "
```

Comment:
This still doesn't seem quite right - maybe `"Invalid status type: %s - must be subclass of stacker.status.Status class." % type(status)` or something similar?